lib/sqlbuilder/builder.go

```diff
@@ -268,10 +272,19 @@ func Map(item interface{}, options *MapOptions) ([]string, []interface{}, error)
 
 		for _, fi := range fieldMap {
 
+			// Check for deprecated tags and give suggestions on how to fix them.
+			deprecatedTags := map[string]string{
+				"stringarray": "postgresql.StringArray",
+				"int64array":  "postgresql.Int64Array",
+			}
+			for k, v := range deprecatedTags {
```

Comment:
1. This is limited to fairly recent lib/pq version (~10 months since initial support, around 6 since the last round of bugfixes) - we should include that in the readme
2. Can we make it a warning and "upgrade" the tags?